aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -393,22 +423,49 @@ The Hosting Bundle installer logs for the module are found at *C:\\Users\\%UserN
 
    * %windir%\SysWOW64\inetsrv\aspnetcore.dll
 
+::: moniker range=">= aspnetcore-2.2"
+
+   * %ProgramFiles%\IIS\Asp.Net Core Module\V2\aspnetcorev2.dll
+
+   * %ProgramFiles(x86)%\IIS\Asp.Net Core Module\V2\aspnetcorev2.dll
+
+::: moniker-end
+
 **IIS Express (x86/amd64):**
 
    * %ProgramFiles%\IIS Express\aspnetcore.dll
 
    * %ProgramFiles(x86)%\IIS Express\aspnetcore.dll
 
+::: moniker range=">= aspnetcore-2.2"
```

Comment:
Should the prior paths (existing in the topic above) be versioned for < 2.2? If not, no worries.